### PR TITLE
Update rename feature for 11 release.

### PR DESCRIPTION
**Pull request by hokein**

### CODE REVIEW

**Review by kirillbobyrev — APPROVED** [review]
Thanks!

**Review by njames93 — COMMENTED** [review]
Isn't it wise to specify that the default behaviour of being enabled is only for clangd-11 and later. We still want the documentation to make sense for people using an older version of clangd. Users of clangd-10 may be confused as to why they aren't getting cross-file rename without specifying the flag, even though the website says it should work.

**Comment by kirillbobyrev** [comment]
uh, I actually didn't think about it, very good point, thanks!

File: features.md [hunk 1]
```diff
@@ -163,14 +163,14 @@ Known limitations
 #### Within-file rename
 {:.v7}
 
-The default mode only allows to rename a local symbol (one which is only used in
-current file).
+This mode only allows to rename a local symbol (one which is only used in
+current file). It is enabled with the command-line flag
+`-cross-file-rename=false`.
```
Comment (kirillbobyrev) [inline features.md:168]: so, basically, do `{:.v11}` and new thing and just keep the existing one as is at the bottom?

File: features.md [hunk 2]
```diff
@@ -163,14 +163,14 @@ Known limitations
...
 
 #### Cross-file rename
 {:.v10}
 
-This mode allows renaming symbols used in several files. It is enabled with the
-command-line flag `-cross-file-rename`.
+This default mode allows renaming symbols used in several files.
```
Comment (kirillbobyrev) [inline features.md:173]: same here.

**Review by kirillbobyrev — CHANGES_REQUESTED** [review]

**Comment by sam-mccall** [comment]
We're about to hit v12. I think the priorities are:

1. document the behavior of current versions
2. hint when older versions might behave differently
3. document the behavior of older versions

If keeping old descriptions around is be invasive to the flow of text, I don't think we should do it - it sacrifices priority 1 to achieve priority 3.

**Pull request merged**